File: C Language/lista10/questao3..c
#include <stdio.h>

int main() {
    int array[5];
    int *p = array;
    printf("Digite 5 valores inteiros:\n");
    for (int i = 0; i < 5; i++) {
        scanf("%d", p + i); 
    }
    printf("\nO dobro de cada valor lido eh:\n");
    for (int i = 0; i < 5; i++) {
        printf("%d\n", *(p + i) * 2); 
    }
    return 0;
}

File: Java Language/ATV 1/questao3/questao3.java
class Questao3 {
	public static void main (String args []) {
		boolean s1, s2, s3;
		int x, y, z;
		x = 3;
		y = ++x;
		z = y++;
		
		s1 = !(x <=y);
		s2 = y == z;
		s3 = (z + 1) <= y;
		
		System.out.println ("O valor de s1 eh: " + s1);
		System.out.println ("O valor de s2 eh: " + s2);
		System.out.println ("O valor de s3 eh: " + s3);
	}
}